cw/cw305/simple_capture_traces.py

```diff
@@ -63,14 +63,21 @@ def plot_results(plot_cfg, project_name):
         print('Project contains no traces. Did the capture fail?')
         return
 
-    # The ADC output is in the interval [-0.5, 0.5). Check that the recorded
-    # traces are within that range with some safety margin.
-    if not (np.all(np.greater(project.waves, -plot_cfg["amplitude_max"]))
-            and np.all(np.less(project.waves, plot_cfg["amplitude_max"]))):
+    # The ADC output is in the interval [-0.5, 0.5) when using doubles or [0, 2**resolution] when
+    # using integers. Check that the recorded traces are within that range with some safety margin.
+    if project.waves[0].dtype == 'uint16':
+        adc_range = np.array([0, 2**12])
```

Comment:
is there a constant stating scope's ADC width so '12' can be replaced by this?

Reply:
Good point, my understanding is that the scope doesn't expose this information and it's also not visible from the waves/np.array. The hardcoded value here is a bit ugly but I didn't see an easy way to avoid it. If we were to switch to `uint8` (Husky would support that) we would at least end up in the `else` condition here and the get a warning as 2**8 > 0.5 (the float limit).

Reply:
Ah, thanks for your input @jpcrypt ! I'll try to rework the PR accordingly to use this.